.github/workflows/publish-sqlfluff-docker-image-to-dockerhub.yaml

```diff
@@ -0,0 +1,65 @@
+# Create and push Docker image of latest release to DockerHub.
+name: Publish SQLFluff DockerHub Version
+on:
+  release:
+    types:
+      - published
+
+# Create tag for integration test.
+env:
+  TEST_TAG: ${{ secrets.DOCKERHUB_USERNAME }}/sqlfluff:test
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      # Get the version of latest release in
+      # order to tag published Docker image.
+      - name: Get latest release name
+        id: latest_release
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: ${{ github.repository }}
+
+      # Setup.
+      - name: Set up QEMU
```

Comment:
Why is this needed? Perhaps for ARM support? Worth mentioning the reason in a comment?

Reply:
Exactly QEMU is the virtualisation software required for building images for multiple platforms. So in our case allows us to build both amd and arm via buildx. I'll add a comment